src/ray/rpc/gcs_server/gcs_rpc_client.h

```diff
@@ -193,6 +193,10 @@ class GcsRpcClient {
   VOID_GCS_RPC_CLIENT_METHOD(HeartbeatInfoGcsService, ReportHeartbeat,
                              heartbeat_info_grpc_client_, )
 
+  /// Check GCS is alive.
+  VOID_GCS_RPC_CLIENT_METHOD(HeartbeatInfoGcsService, CheckAlive,
```

Comment:
Shouldn't `GcsNodeManager` be responsible for checking if a node is alive?

Reply:
This RPC is to check the head is alive.

Reply:
We have met the false reporting issue. So, here comes the improvement.